pkg/deploy/generated/internalclientset/typed/apps/internalversion/generated_expansion.go

```diff
@@ -1,3 +1 @@
 package internalversion
```

Comment:
Delete this file?

Reply:
no, clientgen will recreate it :-) without interface...